en/installconfig-add-nodes.md

```diff
@@ -57,7 +57,7 @@ doesn't work for some reason.
 ## KVM guest nodes
 
 KVM-backed nodes are common and so a little extra guidance is provided here.
-The following actions are performed on the rack controller.
+The following actions are performed on all rack controllers.
```

Comment:
I think this implies all actions *will be* performed on all rack controllers, rather than perhaps *will need to be*?